.github/workflows/docs_bot.yml

```
@@ -0,0 +1,17 @@
+
+name: docs-bot
+on:
+  status
+jobs:
+  run-docs-bot:
+    if: ${{ github.repository == 'apache/tvm' && github.event.state == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Comment link to docs
```

we should skip this on `status`es that go to commits without a PR (e.g. `main`)
```suggestion
      - name: Comment link to docs
         if: github.event.pull_request
```